frontend/src/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import Header from './components/Header.js'
 import Backbone from 'backbone'
 import _ from 'underscore'
-import ExploreHeader from './components/ExploreHeader'
 
-// this is optional really, if we import components directly React will be 
+// this is optional really, if we import components directly React will be
 // in the bundle, so we won't need a global reference
 window.React = React
 window.ReactDOM = ReactDOM
+Backbone.$ = window.$
```

Comment:
This is great. There are big problems with npm-packaged jquery, and this solves that neatly